integration/hammer.go

```diff
@@ -135,30 +150,33 @@ func (pc *pendingCerts) appendCert(submitted *submittedCert) {
 	// Require caller to have checked canAppend().
 	which := 0
 	for ; which < sctCount; which++ {
-		if pc[which] == nil {
+		if pc.certs[which] == nil {
```

Comment:
I was trying to be the least invasive possible, but you're right this is an unexpected sharp edge. Done.